helpers/hpo_panel_matching.py

```diff
@@ -0,0 +1,286 @@
+"""
+all methods relating to:
+- querying metamist for participants and their HPO terms
+- querying PanelApp for all panels with HPO terms
+- loose-matching all HPO terms to relevant panels
+- collecting a list of relevant panels for all participants
+"""
+
+
+import logging
+import json
+import os
+import sys
+import re
+from argparse import ArgumentParser
+from collections import defaultdict
+
+import networkx
+from obonet import read_obo
+from sample_metadata.apis import SeqrApi
+
+from cpg_utils import to_path
+
+from reanalysis.utils import get_json_response
+
+
+MAX_DEPTH: int = 3
+PANELAPP_SERVER = 'https://sample-metadata.populationgenomics.org.au'
+TEMPLATE = (
+    f'{PANELAPP_SERVER}/api/v1/participant/{{dataset}}/individual-metadata-seqr/json'
+)
+
+
+HPO_RE = re.compile(r'HP:[0-9]+')
+OBO_FILE = os.path.join(os.path.dirname(__file__), 'hpo_terms.obo')
```

Comment:
I think this should be altered - currently the process is 100% reliant on an HPO.obo file being committed into the repo, which is ~8MB. Instead re-write this so that the path to the file is an argument